### PR TITLE
Vit

### LINKED ISSUE #3: add support for vision transformers
just trained with masked autoencoding loss; contrastive loss will fall under a separate PR

### PR DESCRIPTION
**Pull request by PPWangyc**
add vit autoencoder; closes #3

### CODE REVIEW

File: beast/models/vits.py [hunk 1]
```diff
@@ -1,13 +1,128 @@
...
+                return latent[:, 0], 0
+        if return_latent:
+            return latent
+        # extract cls latent
+        cls_latent = latent[:, 0] # shape (batch_size, hidden_size)
+        ids_restore = outputs.ids_restore
+        mask = outputs.mask
+
+        decoder_outputs = self.decoder(latent, ids_restore)
+        logits = decoder_outputs.logits  # shape (batch_size, num_patches, patch_size*patch_size*num_channels)
```
Comment (themattinthehatt) [inline beast/models/vits.py:115]: is ViTMAE normalizing the outputs to be in [0, 1]?

**Review by themattinthehatt — CHANGES_REQUESTED** [review]
Looking good! I have some (mostly minor) questions scattered throughout.

Also, can you please run `pip install flake8 isort` and then, from inside the top-level beast repo, run `isort .` (which will automatically organize imports in a consistent way across all files) and `flake8 .` (which will give you a list of PEP style errors to fix; not automatic).
